### PR TITLE
pr changes

### PR DESCRIPTION
**Pull request by Lovepreetcode4each**
1) Chnage the salary struc name to salary structure 
2) add data table in role page 
3)  remove listing word  from side bar

### CODE REVIEW

File: app/Http/Controllers/EmployeeController.php [hunk 1]
```diff
@@ -13,17 +13,130 @@ class EmployeeController extends Controller
...
+                return response()->json([
+                    'data' => $data,
+                    'draw' => request()->input('draw', 1),
+                    'recordsTotal' => $totalRecords,
+                    'recordsFiltered' => $totalRecords,
+                ]);
+            } else {
+                // Query to fetch users with their associated posts, filtered by type_id
+
+                $query = User::with('post')->has('post')->where('type_id', $type);
```
Comment (kumarisavita) [inline app/Http/Controllers/EmployeeController.php:53]: why we are using this post method here?

File: app/Http/Controllers/EmployeeController.php [hunk 2]
```diff
@@ -13,17 +13,130 @@ class EmployeeController extends Controller
...
```
Comment (kumarisavita) [inline app/Http/Controllers/EmployeeController.php:113]: same here ?

File: resources/views/Basicgrade/allBasicGrade.blade.php [hunk 3]
```diff
@@ -25,16 +30,19 @@
...
-       
-            <th scope="row">{{ $val[0]-> grade}} </th>
 
+        <tr>
+            <td scope="row">{{ $val[0]-> grade}} </td>
             <td>
-               <a href="{{ url('edit_grade/'.$val[0]->grade) }}" class="btn btn-primary update" data-id="{{$val[0]->id}}"><i class="bi bi-pencil"></i></a>
-                <a href="" class="btn btn-danger delete" data-id ="{{$val[0]->grade}}"><i class="bi bi-trash"></i></a>
+                <?php
+                $head_title = BasicGradeController::head_title($val[0]->grade);
```
Comment (kumarisavita) [inline resources/views/Basicgrade/allBasicGrade.blade.php:38]: We can call this title value by passing it to view through controller or it's not fusible here. Please explain.

File: resources/views/Employee/salary_structure.blade.php [hunk 4]
```diff
@@ -10,8 +10,7 @@
 
       <div class="card">
         <div class="card-body profile-card pt-4 d-flex flex-column align-items-center">
-
-
+        <label  class="custom-file-upload fas"><div class="img-wrap img-upload"><img for="photo-upload" src="https://github.com/OlgaKoplik/CodePen/blob/master/profile.jpg?raw=true"></div></label>
```
Comment (kumarisavita) [inline resources/views/Employee/salary_structure.blade.php:13]: Is image ko manually download karke rakhlo app apne server per then gave path here.

File: resources/views/Role/role.blade.php [hunk 5]
```diff
@@ -148,7 +148,7 @@
                     <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
                 </div>
                 <div class="modal-body">
-                    Are You want to delete
+                    Are You Sure you  want to delete Role ?
```
Comment (kumarisavita) [inline resources/views/Role/role.blade.php:151]: Make every first letter capital of every word here.

File: resources/views/Salary/allsalaryHead.blade.php [hunk 6]
```diff
@@ -59,7 +59,7 @@
           <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
         </div>
         <div class="modal-body">
-          Are You want to delete
+          Are You  sure You want to delete Head Title ?
```
Comment (kumarisavita) [inline resources/views/Salary/allsalaryHead.blade.php:62]: Same here please make changes like above message.

**Review by kumarisavita — CHANGES_REQUESTED** [review]
Please make changes according to the provided comments.

File: resources/views/Role/role.blade.php (hunk 5 above)
Comment (Lovepreetcode4each) [reply resources/views/Role/role.blade.php:151]: DONE

File: resources/views/Employee/salary_structure.blade.php (hunk 4 above)
Comment (Lovepreetcode4each) [reply resources/views/Employee/salary_structure.blade.php:13]: done

File: resources/views/Basicgrade/allBasicGrade.blade.php (hunk 3 above)
Comment (Lovepreetcode4each) [reply resources/views/Basicgrade/allBasicGrade.blade.php:38]: this  function is call to display  head_title of every grade

File: app/Http/Controllers/EmployeeController.php (hunk 1 above)
Comment (Lovepreetcode4each) [reply app/Http/Controllers/EmployeeController.php]: post replace with user_deatil  to make relationshipbetween user and  user  detail table

File: app/Http/Controllers/EmployeeController.php (hunk 2 above)
Comment (Lovepreetcode4each) [reply app/Http/Controllers/EmployeeController.php]: done

**Review by kumarisavita — APPROVED** [review]

**Pull request merged**